Script/material.dat

```diff
@@ -1827,6 +1873,44 @@ liquid
     AttachedGod = CRUENTUS;
     Effect = EFFECT_MUSTARD_GAS_LIQUID;
   }
+
+  Config OMMEL_BLOOD;
```

Comment:
OK, it seems this material is more trouble that it's worth. I'll think about adding it once it has a place in a quest.